devnet/bridge-terra.yaml

```diff
@@ -0,0 +1,103 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: guardian
+  labels:
+    app: guardian
+spec:
+  ports:
+    - port: 8999
+      name: p2p
+      protocol: UDP
+  clusterIP: None
+  selector:
+    app: guardian
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: guardian
+spec:
+  selector:
+    matchLabels:
+      app: guardian
+  serviceName: guardian
+  replicas: 5
+  updateStrategy:
+    # The StatefulSet rolling update strategy is rather dumb, and updates one pod after another.
+    # If we want blue-green deployments, we should use a Deployment instead.
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: guardian
+    spec:
+      terminationGracePeriodSeconds: 0
+      volumes:
+        - name: key-volume
+          configMap:
+            name: bridge-keypair
+      containers:
+        - name: guardiand
+          image: guardiand-image
+          env:
+            - name: TERRA_FEE_PAYER
+              value: "notice oak worry limit wrap speak medal online prefer cluster roof addict wrist behave treat actual wasp year salad speed social layer crew genius"
+          command:
+# Uncomment this to enable in-place debugging using dlv
+# (not suitable for regular development since the process will no longer restart on its own)
+#
+#            - /dlv
+#            - --listen=:2345
+#            - --accept-multiclient
+#            - --headless=true
+#            - --api-version=2
+#            - --continue=true
+#            - exec
+            - /guardiand
+#            - --
+            - bridge
+            - --ethRPC
+            - ws://eth-devnet:8545
+            - --terraWS
+            - ws://terra-terrad:26657/websocket
+            - --terraLCD
+            - http://terra-lcd:1317
+            - --terraChainID
```

Comment:
Please integrate Terra with the main development workflow vs. having a separate bridge instance. This avoids duplicating code and I don't really see the point in having separate ones.